inventory/group_vars/all.yml

```diff
@@ -3,7 +3,8 @@ ansible_python_interpreter: /usr/bin/python3
 
 app_user: "donalo"
 
-ruby_version: "{{ lookup('file', '../.ruby-version') }}"
+# yamllint disable-line rule:line-length
+ruby_version: "{{ lookup('url', 'https://raw.githubusercontent.com/coopdevs/sharetribe/master/.ruby-version') }}"
```

Comment:
I'm so glad this is open source. I couldn't reach it otherwise. Yet another reason to favor OSS :muscle: